tests/generation/test_generation_utils.py

```diff
@@ -1789,31 +1805,60 @@ def test_contrastive_search_opt(self):
         self.assertListEqual(
             generated_text,
             [
-                """A chat between a curious human and the Statue of Liberty.\n\nHuman: What is your name?\nStatue: I am the Statue of Liberty.\nHuman: Where do you live?\nStatue: New York City.\nHuman: How long have you lived there?\nStatue: Since 1884.\nHuman: Why did you come to America?\nStatue: I was given to the United States by France as a gift for helping the French during the Franco-Prussian War.\nHuman: What do you think of America?\nStatue: I love it. It is the greatest country in the world.\nHuman: What’s the weather like in New York?\nStatue: It is cold.\nHuman: Is it safe to walk around at night?\nStatue: Yes. There are policemen everywhere.\nHuman: Do you have any children?\nStatue: Not yet. My pedestal is empty.\nHuman: What would you like to say to people who want to immigrate to America?\nStatue: Come on over. You will be happy here. We have everything you need.\nSource: http://www.statueofliberty.org/index.cf"""
+                'A chat between a curious human and the Statue of Liberty.\n\nHuman: What is your name?\nStatue: I '
+                'am the Statue of Liberty.\nHuman: Where do you live?\nStatue: New York City.\nHuman: How long have '
+                'you lived there?\nStatue: A hundred years.\nHuman: And you’re from what country?\nStatue: The United '
+                'States of America.\nHuman: Why did you come to America?\nStatue: I came to escape the tyranny of my '
+                'country.\nHuman: What tyranny?\nStatue: They didn’t let me speak my mind.\nHuman: What was your '
+                'country?\nStatue: It was a country of immigrants.\nHuman: Who were the immigrants?\nStatue: They '
+                'were from all over the world.\nHuman: What language did they speak?\nStatue: French, Spanish, '
+                'Italian, German, English—you name it.\nHuman: And where did they come from?\nStatue: They came from '
+                'every country in the world.\nHuman: And you were born in what country?\nStatue: I was born in '
+                'France.\nHuman: And your parents were French?\nStatue'
             ],
         )
 
-    @slow
+    @tooslow
     def test_contrastive_search_gptj(self):
-        article = """DeepMind Technologies is a British artificial intelligence subsidiary of Alphabet Inc. and research laboratory founded in 2010. DeepMind was acquired by Google in 2014. The company is based"""
+        article = (
```

Comment:
on this test:
- Formatted to comply with the 120 char limit
- added the `tooslow` decorator (as with all gptj tests)
- loads the 16bit version of the model, so it can be tested locally more quickly